test_data_conversion_scripts/instructions_lohnsteuer.md

```diff
@@ -0,0 +1,10 @@
+# Converting test data in xlsx from BMF to the YAML file in gettsim tests
+
+## Source of the test data
+The test data is sent out on a mailing list by Referat IA6 together with the new Programmablaufplan. To be added to the mailing list, best contact Steuerrechner@bmf.bund.de
+
+## Instructions for new test data
+1. Save test data for year `year` in `\gettsim-code-for-picking\original_testfaelle` with the name `lohnsteuer_bmf_`year`.xlsx`.
+2. In `convert_lohnst.py` replace the line `lst_data["year"] = `year`` with the correct year.
+3. Run `convert_lohnst.py`. This generates the file `lohnst_converted.csv`.
+4. Run `convert_cvs_tests_to_yamk.py`. This generated the test files in _gettsim_tests in the gettsim repo.
```

Comment:
```suggestion
4. Run `convert_cvs_tests_to_yaml.py`. This generated the test files in _gettsim_tests in the gettsim repo.
```
Probably need to say something about the folder structure here? Or do you look for the installation location of GETTSIM based on the import?